spec/features/gpl207_bug_incorrect_submission_id_on_transfer.rb

```diff
@@ -0,0 +1,202 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Bug research' do
```

Comment:
Did you intend to leave this saying  'Bug research'? Feel like it should describe more accurately what the test i